tests/tests.go

```diff
@@ -33,7 +33,7 @@ const (
 )
 
 var (
-	gracePeriodSeconds int64 = 10
+	gracePeriodSeconds int64 = 3
```

Comment:
Why are you lowering this?

Reply:
just to make the tests run faster we don't need to wait 10 sec